readthedocs/projects/views/private.py

```diff
@@ -506,19 +506,25 @@ def project_notifications(request, project_slug):
     project = get_object_or_404(
         Project.objects.for_admin_user(request.user), slug=project_slug)
 
-    email_form = EmailHookForm(data=request.POST or None, project=project)
-    webhook_form = WebHookForm(data=request.POST or None, project=project)
+    project_dashboard = reverse('projects_notifications', args=[project.slug])
 
-    if request.method == 'POST':
+    if request.method == 'POST' and 'email' in request.POST:
+        email_form = EmailHookForm(data=request.POST or None, project=project)
```

Comment:
We don't need the `or None` part here, since we already checking if the method is POST